prow/cmd/peribolos/main_test.go

```diff
@@ -698,6 +698,13 @@ func (c *fakeTeamClient) EditTeam(team github.Team) (*github.Team, error) {
 	if team.Privacy != "" {
 		t.Privacy = team.Privacy
 	}
+	if team.ParentTeamID != nil {
+		team.Parent = &github.Team{
```

Comment:
Wrong lvalue

Reply:
I'm confused.. I don't think it is. The goal here in the fake client is to take ParentTeamID, and set the Parent struct to contain a fake team with that matching ID.

Reply:
Agreed, and what happens to `team.Parent` when this method returns? Compare the lvalue in this `if` statement to the others.